docs/themes.md

```diff
@@ -1,7 +1,7 @@
 # Themes
 
 ```sh
-npm i mdx-deck/themes
+npm i mdx-deck
```

Comment:
I meant, I don't think it makes sense to repeat the install instructions here since it's now the same package. I think this entire code fence can be removed

Reply:
Aha, I missed that! The entire code-fenced block's now been removed in my latest commit.